libs/com.etendorx.utils.auth/src/main/java/com/etendorx/utils/auth/key/JwtKeyUtils.java

```diff
@@ -151,12 +161,17 @@ public static void validateTokenValues(Map<String, Object> tokenValuesMap, List<
     }
   }
 
-  public static Claims generateUserClaims(String userId, String clientId, String orgId) {
+  public static Claims generateUserClaims(String userId, String clientId, String orgId, String defaultOrgId,
+                                          String defaultRoleId, String searchKey, String serviceId) {
     Claims claims = new DefaultClaims();
     claims.put(USER_ID_CLAIM, userId);
     claims.put(CLIENT_ID_CLAIM, clientId);
-    claims.put(ORG_ID_CLAIM, orgId);
+    //claims.put(ORG_ID_CLAIM, orgId);
```

Comment:
What is the reason for having orgId and defaultOrgId? In the case of token creation it should only contain orgId, roleId, etc. ... it does not apply to have default data.